Cargo.toml

```diff
@@ -32,11 +32,19 @@ thiserror = "1.0"
 askama = "0.10"
 
 # This only supports up to 0.7. See examples/actix/main.rs for more.
-actix = "0.9"
+actix = "0.10"
 actix-web = { version = "2.0", features = ["openssl"] }
 actix-session = "0.3"
 actix-files = "0.2"
 
+# Example, use the version numbers you need
+tide = "0.15.0"
+tide-rustls = "0.1.3"
```

Comment:
In versions I think it's best not to hardcode the patch release so `tide-rustls = "0.1.3"` should be `tide-rustls = "0.1"` so that cargo update works when new fixes come out. Similar for the other added deps.